### ISSUE TITLE
Show time correctly

### ISSUE THREAD

**Issue by enoch85** [owncloud/calendar#877]
I propose a change for this:

*Now* 
[Imgur](http://i.imgur.com/LUoSFmS.png)

*Change*
http://i.imgur.com/gdlMss8.png

The reason is obvious, you want to be able to see the end time and not just the start time.

**Comment by georgehrke** [comment]
cc @jancborchardt

**Comment by georgehrke** [comment]
I'm not sure about this change. If you want a detailed schedule for the upcoming days, you can just check the week view. I don't think we need this in the month view.

**Comment by enoch85** [comment]
@georgehrke No, I use month view all the time, it gives me a good overview without having to scroll (like I have to do in week view) to see anything. 

For instance, if I have a long workday I have to scroll down to see the end. With this fix I wouln't have to to that. It's not a big change.

Another suggestion is to show the time as a tooltip in month view.

**Comment by georgehrke** [comment]
The end-time should definitely be shown in the tooltip, that's where I agree

**Comment by enoch85** [comment]
Or move the time totally to a tooltip? So instead of showing the start time, show the start *and* the end time in the tooltip.

**Comment by jancborchardt** [comment]
Well, the start time should be shown outright of course. Details can be shown in the tooltip.

Please make sure we also do this for Gallery+, cc @libasys

**Comment by enoch85** [comment]
@jancborchardt So you think that this is too much?

<a href="http://imgur.com/ck25wj9"><img src="http://i.imgur.com/ck25wj9.png" title="source: imgur.com" /></a>

**Comment by jancborchardt** [comment]
Should be `10:00 – 12:00`, eh? The tooltip should also show the full title of the event of course because that is likely to be cut off in the month view.

**Comment by enoch85** [comment]
@jancborchardt Eh, good point. ;)

**Comment by IKevin90** [comment]
when does the tooltip come?

**Comment by jancborchardt** [comment]
@IKevin90 when someone implements it. You’re welcome to.
